preupg/cli.py

```diff
@@ -54,126 +54,166 @@ def __init__(self, args=None):
-            help="Select mode which can be used for migration or upgrade"
+            help="Select what you plan to do with the system after performing"
+                 " its assessment by Preupgrate Assistant - migration or"
+                 " upgrade. Both modes are selected by default. This option"
+                 " may only affect behaviour of the modules - they can provide"
+                 " different results when only one mode is selected. Use one"
+                 " of these values: migrate, upgrade. It may be that modules"
+                 " behave the same no matter what mode is selected."
         )
         self.parser.add_option(
-            "--kickstart",
+            "-k", "--kickstart",
             action="store_true",
             default=False,
-            help="Generate kickstart"
+            help="Generate kickstart file that is to be used for migration."
+                 " Preupgrade Assistant system assessment needs to be"
+                 " performed before using this option. You may use the"
+                 " preupg-kickstart-generator tool instead of this option -"
+                 " it does the same."
         )
         self.parser.add_option(
             "--select-rules",
             metavar="RULES",
-            help="Run a set of contents give as a comma-separated list."
+            help="Execute just a subset of modules out of a module set."
+                 " Multiple modules are to be separated by a comma." +
+                 " " * 5 +
+                 "Example:  --select-rules xccdf_preupg_rule_networking_vsftpd"
+                 "_check,xccdf_preupg_rule_networking_bind_configuration_check"
         )
         self.parser.add_option(
-            "--list-rules",
+            "-L", "--list-rules",
             action="store_true",
             default=False,
-            help="List available contents"
+            help="List all the modules available within a module set."
         )
         self.parser.add_option(
-            "--version",
-            action="store_true",
-            default=False,
-            help="Show Preupgrade Assistant version"
-        )
-        self.parser.add_option(
-            "--dst-arch",
+            "-D", "--dst-arch",
             metavar="DSTARCH",
-            help="""
-                 Specify target/destination architecture where you want to migrate
-                 (e.g.: x86_64). Use of the option is expected on 32b systems.
-                 """
+            help=(
+                "Specify an architecture of the system to be migrate to."
+                " Available option are: %s. Use of the option is expected on"
+                " 32-bit systems as by the release of RHEL 7, 32-bit hardware"
+                " support has been dropped."
+                % ", ".join(settings.migration_options)
+            )
         )
         self.parser.add_option(
-            "--old-report-style",
+            "-o", "--old-report-style",
```

Comment:
-o is usually used for output file. As somewhere was requested fueature to specify where should be report file - or kickstart? - stored, it will not be intuitive. I prefer keep here only long form of the option.